### PR TITLE
docs: fix docs and rendering issues with index.mdx

### PR DESCRIPTION
**Pull request by swaptr**
This hook will update `docs/index.mdx` if there were changes in `README.md.
Note: Merge after #78 for a cleaner commit history :)

### CODE REVIEW

**Review by vincenzopalazzo — COMMENTED** [review]
Good thanks!

There is only a change that you need to make in the other PR

**Comment by swaptr** [comment]
Wait I need to make the `hook` accessible. How can I do that?

**Comment by vincenzopalazzo** [comment]
>Wait I need to make the hook accessible. How can I do that?

What type of hooks are you talking about?

**Comment by swaptr** [comment]
I am talking about git hooks.

**Comment by vincenzopalazzo** [comment]
Ignore it, we have more important stuff to do

File: docs/index.mdx [hunk 1]
```diff
@@ -1,26 +1,43 @@
----
-description: ":dart: Dart framework for Core Lightning :dart: "
----
+<div align="center">
+  <h1>lndart.cln</h1>
+
+  <img src="https://github.com/dart-lightning/icons/raw/main/main/res/mipmap-xxxhdpi/ic_launcher.png" />
+
+  <p>
+    <strong> :dart: Dart framework for Core Lightning :dart: </strong>
```
Comment (vincenzopalazzo) [inline docs/index.mdx:7]:
```suggestion
    <strong>  Dart framework for Core Lightning  </strong>
```

File: docs/index.mdx [hunk 2]
```diff
@@ -1,26 +1,43 @@
...
+  </p>
 
-<img src="https://github.com/dart-lightning/icons/raw/main/main/res/mipmap-xxxhdpi/ic_launcher.png" />
+  <p>
+   <img alt="GitHub Workflow Status" src="https://img.shields.io/github/workflow/status/dart-lightning/clightning.dart/Sanity%20Check?style=flat-square">
```
Comment (vincenzopalazzo) [inline docs/index.mdx:11]:
```suggestion
   <img alt="GitHub Workflow Status" src="https://img.shields.io/github/workflow/status/dart-lightning/clightning.dart/Sanity%20Check?style=flat-square"/>
```

This should be fix the docs page

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]

File: docs/index.mdx (hunk 2 above)
Comment (vincenzopalazzo) [reply docs/index.mdx:11]: This is not magic, I use the https://mdxjs.com/playground/ and it show the error and the line

**Review by vincenzopalazzo — APPROVED** [review]

**Review by vincenzopalazzo — APPROVED** [review]

**Pull request merged**